lib/common/client/concerns/log_as_warning_helpers.rb

```diff
@@ -1,19 +0,0 @@
-# frozen_string_literal: true
-
-module Common::Client
-  module Concerns
-    module LogAsWarningHelpers
-      def warn_for_service_unavailable
```

Comment:
correct! I think I would have missed this except some specs were failing to run because of the error `uninitialized constant Common::Client::Concerns::LogAsWarningHelpers` after I had deleted lib/emis/service.rb.